packages/vc-api-issuer/__tests__/shared/vcApiIssuerAgentLogic.ts

```diff
@@ -13,7 +13,7 @@ export default (testContext: { getAgent: () => ConfiguredAgent; setup: () => Pro
     })
 
     afterAll(async () => {
-      await new Promise<void>((resolve) => setTimeout(() => resolve(), 5000)) // avoid jest open handle error
+      await new Promise<void>((resolve) => setTimeout(() => resolve(), 10000)) // avoid jest open handle error
       await testContext.tearDown()
```

Comment:
Interesting trick

Reply:
Gives you a certain timeout after everything finishes indeed